package.json

```diff
@@ -6,17 +6,18 @@
   "scripts": {
     "prepublish": "npm run test && coffee -c lib",
     "lint": "resin-lint lib",
-    "test": "npm run lint"
+    "test": "npm run lint",
+    "versionist": "versionist"
   },
   "author": "Petros Angelatos",
   "license": "Apache-2.0",
   "devDependencies": {
     "coffee-script": "^1.10.0",
-    "resin-lint": "^1.2.2",
-    "mocha": "^2.4.5",
     "gulp": "^3.9.0",
     "gulp-coffee": "^2.2.0",
-    "gulp-util": "^3.0.6"
+    "gulp-util": "^3.0.6",
+    "resin-lint": "^1.2.2",
+    "versionist": "^2.6.2"
```

Comment:
versionist `2.7.0` has been published and it should fall back to its internal `versionist.conf.js` by default